doc: fast_pair: Allow hiding UI indication (#7929)
nrfconnect/sdk-nrf: samples/bluetooth/peripheral_fast_pair/README.rst

=======

Button 1:
   Toggles between three Fast Pair advertising modes:

   * Fast Pair discoverable advertising.

>    * Fast Pair not discoverable advertising, show UI indication.

Reviewer: ```suggestion
   * Fast Pair not discoverable advertising (with the show UI indication).
```

   * Off if there is no Central connected.

LED 3:
   Depending on the Fast Pair advertising mode setting:

   * On if the device is Fast Pair discoverable.

>    * Blinks with 0.5 secs interval if the selected mode is Fast Pair not discoverable advertising, show UI indication.

Reviewer: ```suggestion
   * Blinks with 0.5 secs interval if the selected mode is the Fast Pair not discoverable advertising with the show UI indication.
```

>    * Blinks with 1.5 secs interval if the selected mode is Fast Pair not discoverable advertising, hide UI indication.

Reviewer: ```suggestion
   * Blinks with 1.5 secs interval if the selected mode is the Fast Pair not discoverable advertising with the hide UI indication.
```

   a. Go to :guilabel:`Settings` > :guilabel:`Google` > :guilabel:`Devices & sharing` (or :guilabel:`Device connections`) > :guilabel:`Devices` > :guilabel:`Saved devices`.
   #. Verify that the paired device is appearing on the list.

#. Press **Button 1** to switch to the Fast Pair not discoverable advertising show UI indication mode.

>    **LED 3** blinks rapidly.

Reviewer: ```suggestion
   **LED 3** starts blinking rapidly.
```

   If you want to test the Fast Pair not discoverable advertising hide UI indication mode, press **Button 1** again.

>    **LED 3** blinks slowly then.

Reviewer: ```suggestion
   **LED 3** starts blinking slowly.
```

#. Wait until the Fast Pair Provider is added to :guilabel:`Saved devices` on the second Android device:

   a. Go to :guilabel:`Settings` > :guilabel:`Google` > :guilabel:`Devices & sharing` (or :guilabel:`Device connections`) > :guilabel:`Devices` > :guilabel:`Saved devices`.
      The paired device appears on the list.
   #. If the device does not appear on the list, wait until the data is synced between phones.

#. Move the second Android device close to the Fast Pair Provider.

>    If the device is in the show UI indication mode, a notification similar to the following one appears:

Reviewer: ```suggestion
   If the device is in the show UI indication advertising mode, a notification similar to the following one appears:
```

   .. figure:: /images/bt_fast_pair_not_discoverable_notification.png
      :scale: 50 %
      :alt: Fast Pair not discoverable advertising Android notification

>    If the device is in the hide UI indication mode, no notification appears.

Reviewer: ```suggestion
   If the device is in the hide UI indication advertising mode, no notification appears.
```

   The device advertises (you can check that in the nRF Connect for Mobile app), but do not want to be paired with.

> #. If the notification appears, tap on it to trigger the Fast Pair procedure.

Reviewer: ```suggestion
#. In the show UI indication mode, when the notification appears, tap on it to trigger the Fast Pair procedure.
```
